src/dvc_render/vega_templates.py

```diff
@@ -508,23 +501,46 @@ class SmoothLinearTemplate(Template):
                     },
                     "color": {"field": "rev", "type": "nominal"},
                 },
-                "layer": [
+                "transform": [
                     {
-                        "mark": {
-                            "type": "line",
-                            "point": True,
-                            "tooltip": {"content": "data"},
-                        }
-                    }
+                        "loess": Template.anchor("y"),
+                        "on": Template.anchor("x"),
+                        "groupby": ["rev", "filename::field"],
+                        "bandwidth": {"signal": "smooth"},
+                    },
                 ],
-            }
+            },
+            {
+                "mark": {
+                    "type": "point",
+                    "tooltip": {"content": "data"},
+                },
+                "encoding": {
+                    "x": {
+                        "field": Template.anchor("x"),
+                        "type": "quantitative",
+                        "title": Template.anchor("x_label"),
+                    },
+                    "y": {
+                        "field": Template.anchor("y"),
+                        "type": "quantitative",
+                        "title": Template.anchor("y_label"),
+                        "scale": {"zero": False},
+                    },
+                    "color": {"field": "rev", "type": "nominal"},
+                },
+            },
         ],
     }
 
 
-class LinearTemplate(Template):
+class LinearTemplate(SmoothLinearTemplate):
     DEFAULT_NAME = "linear"
 
+
+class SimpleLinearTemplate(Template):
```

Comment:
Made the smooth template the default here since it shows points, and added back a simple template without points or smoothing. We can update the sklearn plots to use the simple template.